docs/setup/apache.md

```diff
@@ -115,6 +115,10 @@ AddType application/vnd.ogc.context+xml .wmc
 
 ErrorDocument 502 /errors/50x.html
 ErrorDocument 503 /errors/50x.html
+
+# redirects onto the viewer by default
+RewriteRule ^/$ https://%{SERVER_NAME}/mapfishapp/ [R=301,L]
```

Comment:
This is handled by the SP IIRC
https://github.com/georchestra/datadir/blob/6eb3dd3f3c09098804a1728de3aa493480a2ce4a/security-proxy/security-proxy.properties#L131-L134

Reply:
by default, I had a directory listing from apache2. If nothing handles the '/', apache2 will handle it.

But for sure, if you hit the SP directly, it will suggest a redirection to the default target.